api-catalog-ui/frontend/src/components/Wizard/configs/wizard_node_categories.jsx

```diff
@@ -41,33 +45,40 @@ export const nodeSpecificCategories = [
             host: {
                 value: '',
                 question: 'The host to be used:',
+                tooltip: 'e.g. localhost',
             },
             ipAddress: {
                 value: '',
                 question: 'The IP address to be used:',
+                tooltip: 'e.g. 127.0.0.1',
             },
             port: {
                 value: '',
                 question: 'The port to be used:',
+                tooltip: 'e.g. 10011',
             },
             servicePath: {
                 value: '',
                 question: 'The service path:',
+                tooltip: 'e.g. /eureka/apps/',
             },
             maxRetries: {
                 value: '',
                 question: 'The maximum number of retries:',
                 hide: true,
+                tooltip: 'e.g. 30',
             },
             requestRetryDelay: {
                 value: '',
                 question: 'The request retry delay:',
                 hide: true,
+                tooltip: 'e.g. 1000',
             },
             registryFetchInterval: {
                 value: '',
                 question: 'The interval for registry interval:',
                 hide: true,
+                tooltip: 'e.g. 5',
```

Comment:
Add additional information, something like: `How often does Eureka client pull the service list from Eureka server. The default is 30 seconds. Example: 5`